js/actions/actions.js

```diff
@@ -21,27 +22,33 @@ module.exports = {
     var message = {
       text: text,
       timestamp: timestamp,
-      user: 'guest' // hard-code for now
+      user: 'guest', // hard-code for now
+      clientId: UUID.generate()
     };
 
+    dispatch(ActionTypes.NEW_MESSAGE, message);
```

Comment:
can we call this CREATE_MESSAGE or CREATE_MESSAGE_PENDING, so that all 3 are like a family?